src/components/auth/signup/Signup.jsx

```diff
@@ -0,0 +1,107 @@
+import GoogleButton from '../../../assets/images/google-button.png';
+import SlackButton from '../../../assets/images/slack-button.png';
+
+export default class Signup extends Component {
+  componentDidMount() {}
+
+  render() {
+    return (
+      <StyledSignup>
+        <form
+          className="text-center border border-light p-5 w-50 z-depth-1"
+          action="#!"
+        >
+          <p className="h4 mb-4">Sign in</p>
+
+          <label htmlFor="username" className="d-flex font-weight-bold">
+            Username
+          </label>
+          <input
+            type="text"
+            id="username"
+            className="form-control mb-4"
+            placeholder="Enter Username"
+          />
+
+          <label htmlFor="email" className="d-flex font-weight-bold">
+            Email
+          </label>
+          <input
+            type="email"
+            id="email"
+            className="form-control mb-4"
+            placeholder="E-mail"
+          />
+
+          <div className="d-flex justify-content-between">
+            <div className="passwordContainer">
+              <label htmlFor="password" className="d-flex font-weight-bold">
+                Password
+              </label>
+              <input
+                type="password"
+                id="password"
+                className="form-control mb-4"
+                placeholder="Password"
+              />
+            </div>
+
+            <div className="passwordContainer">
+              <label
+                htmlFor="confirmPassword"
+                className="d-flex font-weight-bold"
+              >
+                Confirm Password
+              </label>
+              <input
+                type="password"
+                id="confirmPassword"
+                className="form-control mb-4"
+                placeholder="Confirm Password"
+              />
+            </div>
+          </div>
+
+          <div className="d-flex justify-content-around" />
+
+          <button className="btn btn-info btn-block my-4" type="submit">
+            Sign Up
+          </button>
+
+          <div className="dividerContainer">
+            <div className="divider">
+              <hr />
+            </div>
+            <p className="font-weight-bold">OR</p>
+            <div className="divider">
+              <hr />
+            </div>
+          </div>
+
+          <div className="d-flex">
+            <div className="optional-login">
+              <a href="##">
+                <img src={GoogleButton} alt="Sign up with google" />
+              </a>
+            </div>
+
+            <div className="optional-login">
+              <a href="##">
+                <img src={SlackButton} alt="Sign up with slack" />
+              </a>
+            </div>
+          </div>
+
+          <p>
+            Already have an account? &nbsp;
+            <Link to="/login">Sign In</Link>
+          </p>
+        </form>
+      </StyledSignup>
+    );
+  }
+}
```

Comment:
I can't seem to find the processing of the form inputs, is that beyond the scope of this PR?

Reply:
Yes, it beyond the scope of the PR